deployments/ticdc/docker/kafka-consumer.Dockerfile

```diff
@@ -1,9 +1,12 @@
+# syntax = docker/dockerfile:1-experimental
+
 FROM golang:1.20-alpine as builder
 RUN apk add --no-cache make bash
 WORKDIR /go/src/github.com/pingcap/tiflow
 COPY . .
 
-RUN make kafka_consumer
+RUN --mount=type=cache,target=/go/pkg/mod go mod download
```

Comment:
the target path is `$GOPATH` specific, it can't guarantee that `$GOPATH` is `/go` in each environment.

Reply:
I misunderstood the `target`, seems ok